app/views/citizens/income_summary/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <%= page_template page_title: t('.page_heading'), template: :form do %>
 
-  <p class="govuk-body"><%= t('.add_all_income') %></p>
+  <h2 class="govuk-heading-m">
+    <%= t(
+          '.add_all_income',
+          start_at: l(@legal_aid_application.transaction_period_start_at.to_date, format: :long_date),
+          finish_at: l(@legal_aid_application.transaction_period_start_at.to_date, format: :long_date)
```

Comment:
This feedback's a bit late, sorry! I'm only looking at this because there's a similar change to be made to the outgoing payments screen and wanted to get an idea how to approach it, but I spotted a small bug.

This line is displaying the `transaction_period_start_at` date when it should be the `transaction_period_finish_at` date, ie:

```finish_at: l(@legal_aid_application.transaction_period_finish_at.to_date, format: :long_date)```

Reply:
Good spot - thank you